src/exit.rs

```diff
@@ -133,10 +133,36 @@ pub enum AxVCpuExitReason {
     },
     /// The vcpu is halted.
     Halt,
+    /// Try to bring up a secondary CPU.
+    ///
+    /// This is used to notify the hypervisor that the target vcpu
+    /// is powered on and ready to run, generally used in the boot process
+    /// of a multi-core VM.
+    /// This VM exit reason is architecture-specific, may be triggered by
+    /// * a PSCI call in ARM
+    /// * a SIPI in x86
+    /// * a sbi call in RISC-V
+    CpuUp {
```

Comment:
Should we give this variant a better name? `CpuUp` is a little bit confusing, as there is another variant named `CpuDown` but these two variants are actually not directly related as their names suggest.

Reply:
What do you mean

How about `CpuOn` & `CpuOff`

I think CpuUp looks good.

ref: ArceOS's psci [impl](https://github.com/arceos-org/arceos/blob/f03dcbba5400cb911cd950ea60447f0690007618/modules/axhal/src/platform/aarch64_common/psci.rs#L112)

![image](https://github.com/user-attachments/assets/9b27f2e9-877c-4eb7-bd79-edcde4459a1a)

[hart_start & hart_stop](https://github.com/rustsbi/rustsbi/blob/b32d35ffa84ab840e695e7f307715c9a3d3e7ef8/sbi-rt/src/hsm.rs#L61) in RustSBI